src/main/resources/db/migration/common/V1_312__insert_automated_task_activity.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO audit_activity (aua_id, activity_name, activity_description, created_ts, created_by, last_modified_ts, last_modified_by)
+VALUES (nextval('aua_seq'), 'Run Job Manually', 'Run Job Manually', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0);
+INSERT INTO audit_activity (aua_id, activity_name, activity_description, created_ts, created_by, last_modified_ts, last_modified_by)
+VALUES (nextval('aua_seq'), 'Enable/Disable Job', 'Enable/Disable Job', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0);
```

Comment:
Can you ensure these are added to the modernised_data_standing_data.sql script please?

Reply:
np